code-challenge/src/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import { Provider } from 'react-redux';
+import { store } from './redux/createStore';
+
+ReactDOM.render(
+    <Provider store={store}>
+        <App />
+    </Provider>,
+  document.getElementById('root')
```

Comment:
Curious, did you remove `React.StrictMode` or was it not auto-generated?

Reply:
I removed it during my quick copy/paste of Provider honestly